src/pages/components/content-block-with-cards.mdx

```diff
@@ -37,9 +38,22 @@ Sometimes a video will support the pages' narrative, but should not be prominent
 
 ![Image of Content block simple with aside elements](../../images/pattern/content-block-with-cards/content-block-with-cards-with-videos.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) | Notes | 
+| -------- | --------- | ---------- | ---------- | ----------| ----------|
+| Headline | Short copy | Yes | 1 | XXS (40 / 55)| - |
+| TOC headline override | Short copy | No | 1 | Mini (25 / 35)| - |
+| [Card](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/card) | Component | Yes | 2-6 | - | You can use the simple card, image card, or video card, but all cards must be of the same type. |
+| [CTA (card)](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/cta) | Component | No | 1 | - | - |
+
+Note: This uses the underlying component "Card group". You can find more information about that component <a href="https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/card-group">here</a>.
```

Comment:
Is there a reason we call it out here instead of up above? Just curious...

Reply:
I'm updating this to just be card group